brian2cuda/cuda_prefs.py

```diff
@@ -38,3 +55,145 @@
         validator=lambda v: v is None or isinstance(v, str)
     ),
 )
+
+# Preferences
+prefs.register_preferences(
+    'devices.cuda_standalone',
+    'CUDA standalone preferences',
+
+    SM_multiplier = BrianPreference(
+        default=1,
+        docs='''
+        The number of blocks per SM. By default, this value is set to 1.
+        ''',
+        ),
+
+    parallel_blocks = BrianPreference(
+        docs='''
+        The total number of parallel blocks to use. The default is the number
+        of streaming multiprocessors.
+        ''',
+        validator=lambda v: v is None or (isinstance(v, int) and v > 0),
+        default=None),
+
+    gpu_heap_size = BrianPreference(
```

Comment:
This should go into `cuda_backend`

Reply:
Should I register a new preference block called `cuda_backend` or put it in `brian_backend`?

Reply:
Woops, I wrote that wrong. It should all be in `cuda_backend`, there should be no `brian_backend`. Sorry about that, can you change that?